pkgs/top-level/perl-packages.nix

```diff
@@ -822,9 +862,9 @@ let
       shortenPerlShebang $out/bin/cpm
     '';
     meta = {
-      homepage = "https://github.com/skaji/cpm";
-      description = "A fast CPAN module installer";
+      description = "a fast CPAN module installer";
```

Comment:
The same for `description`: it is litelly from CPAN.
I tried to upcase the first letter, but that result in dozens of upcasing diffs, when `nixpkgs`'s description is lowcased.
So I ended up with keep it as in CPAN (just trimming spaces and ending dot if any)

Reply:
also, upcasing the first letter of `description = "tcp_connect with transparent proxy handling";` seemed wrong